frontend/public/components/nav/items.tsx

```diff
@@ -100,6 +100,7 @@ export type NavLinkProps = {
   disallowed?: string;
   startsWith?: string[];
   activePath?: string;
+  mergeAfter?: string;
```

Comment:
This shouldn't be necessary, Console `NavList` has no use for this prop.

Reply:
ok, done